src/Azs.Compute.Admin/readme.md

```diff
@@ -71,6 +74,12 @@ directive:
     set:
       subject-prefix: Compute
 
+    # Prepend Compute for the Quota cmdlets
+  - where:
```

Comment:
Remove this

Reply:
- where:
      verb: Get
      subject: ComputeScaleUnit
    remove: True

Reply:
I updated with the remove: True. Can you take a look?